artifactory/federation.go

```diff
@@ -33,17 +33,24 @@ type MirrorLags struct {
 func (c *Client) FetchMirrorLags() (MirrorLags, error) {
 	var mirrorLags MirrorLags
 	c.logger.Debug("Fetching mirror lags")
+
 	resp, err := c.FetchHTTP(federationMirrorsLagEndpoint)
 	if err != nil {
-		if err.(*APIError).status == 404 {
```

Comment:
Replacing the construction from line 38 with construction in line 41 pleases me very much. Full support from me!